zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java

```diff
@@ -136,6 +136,8 @@ public Object call() throws Exception {
           logger.info("Ran runKerberosLogin command successfully.");
           kinitFailCount = 0;
           // schedule another kinit run with a fixed delay.
+          logger.info("Scheduling Kerberos ticket refresh thread with interval " +
```

Comment:
Could you help to rename `logger` to `LOGGER` ? It is our convention for naming static variables.

Reply:
The keyword "final" is also desired.

Reply:
Done.
1. logger is replaced with LOGGER
2. LOGGER is now declared final